pydal/adapters/mongo.py

```diff
@@ -8,6 +8,8 @@
 from ..helpers.classes import SQLALL, Reference
 from ..helpers.methods import use_common_filters, xorify
 from .base import NoSQLAdapter
+from bson import Binary
```

Comment:
mind that such imports raise troubles to users that don't have pymongo/bson installed